ci-operator/config/redhat-developer/openshift-jenkins-operator/redhat-developer-openshift-jenkins-operator-master.yaml

```diff
@@ -0,0 +1,41 @@
+base_images:
+  base:
+    name: "4.2"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
+images:
+- from: base
+  to: openshift-jenkins-operator
+promotion:
+  name: "4.2"
+  namespace: ocp
```

Comment:
does this actually need to be promoted?  what is the expected publication mechanism for this operator?

tagging it for promotion is ok, but it's one step away from getting it included in the OCP release payload which i do not think we want.

Reply:
(and since your tests aren't actually making use of the promoted image currently, i don't think you need to do so)

Reply:
OK, I removed it